src/foam/core/Boot.js

```diff
@@ -192,28 +192,49 @@ foam.LIB({
       /* Creates a Foam class from a plain-old-object definition.
           @method CLASS
           @memberof module:foam */
-      foam.CLASS = function(m) {
-        var cls   = m.class ? foam.lookup(m.class) : foam.core.Model;
-        var model = cls.create(m);
-        model.validate();
-        var cls = model.buildClass();
+      foam.CLASS = function(clsDef) {
```

Comment:
Why the need to rename everything? Makes the change look artificially larger than it is and conflates two things.